gems/smithy/spec/fixtures/service_index/model.smithy

```diff
@@ -91,3 +91,22 @@ structure OrphanedStructure {}
 
 @error("client")
 structure OrphanedError {}
+
+operation RecursiveShapesOperation {
```

Comment:
Probably can add a recursive service with just a simple operation and structure that references itself, then other indexes can use this same fixture. I can do that later if you want.